### ISSUE TITLE
Could not load libcrypto.dll

### ISSUE THREAD

**Issue by juancarlospaco** [dom96/choosenim#211]
```powershell
PS C:\Users\user> .\choosenim.exe
Error: Could not load: (libcrypto-1_1-x64|libeay64).dll

PS C:\Users\user> 
```

Windows 10, x86_64, fresh install, `0.6.0` [EXE from GitHub release.](https://github.com/dom96/choosenim/releases/tag/v0.6.0)

I can try to find the DLL manually for myself, but would be nice to have this with better UX for end users.

Maybe provide a statically compiled alternative binary.

**Comment by dom96** [comment]
Download the zip not the exe?

**Comment by juancarlospaco** [comment]
I know is doable, I am talking about UX.
Nim can compile statically, just for this binary, not everything.
Should be possible to make a Powershell one-liner to download choosenim and run it, just like the one-liner for Linux/Mac.

**Comment by genotrance** [comment]
This is fixed in #225.
